examples/sotest/modprint/modprint.c

```diff
@@ -57,7 +40,8 @@ static const struct symtab_s g_modprint_exports[1] =
  * Name: modprint
  ****************************************************************************/
 
-static void modprint(FAR const char *fmt, ...)
+__attribute__((visibility("default")))
```

Comment:
See comments for __attribute((destructor)).  This applies to the many occurrences of the visibility attribute.